src/Animations/animationGroup.ts

```diff
@@ -21,6 +21,19 @@ export class TargetedAnimation {
      * Target to animate
      */
     public target: any;
+
+    public constructor(animation: Animation, target: any){

```

Comment:
that is a major breaking change :)

Reply:
Can you help me find a better way? I only added the constructor due to how we initialize TargetAnimation via:
https://github.com/BabylonJS/Babylon.js/pull/6333/files#diff-10c0fe63a65cb9102072d2577dc27c04L180

which doesn't seem to work now that I added serialize().

Reply:
Ah, i think i just wanted to be fancy with field initialization, calling the default constructor should work, right?